### DISCUSSION TITLE
Creating thread via command

### DISCUSSION THREAD

**Discussion by civilCornball** [discord-php/DiscordPHP#1209] [Q&A] [answered]
I'm attempting to have a bot create threads in a "forum."

I found [a relevant part of the Discord documentation](https://discord.com/developers/docs/resources/channel#start-thread-in-forum-or-media-channel), which pointed me towards having my bot make a thread.

But I can't find a command that'd do that.

The closest I can get is **Thread::sendMessage()**, documentation found [here](https://discord-php.github.io/DiscordPHP/classes/Discord-Parts-Thread-Thread.html#method_sendMessage).

Any help? An example would be especially appreciated.

([Though this one looks similar, they already have a thread to post to.](https://github.com/discord-php/DiscordPHP/discussions/1084))

**Comment by SQKo** [comment, marked as answer]
The method you are looking for is `Channel::startThread()` in the documentation: https://discord-php.github.io/DiscordPHP/classes/Discord-Parts-Channel-Channel.html#method_startThread available since v10

After thread is created in forum, you can also send a message using `Thread::sendMessage()`

Lacking in documentation, here is the example:
```php
// Start thread (Channel type must be forum)
$channel->startThread([
    'name' => 'New forum post',
    'message' => MessageBuilder::new()->setContent('Post content here'),
    // more options here from the relevant discord documentation that you found
])->then(function (Thread $thread) {
    // make another message
    return $thread->sendMessage(MessageBuilder::new()->setContent('A post reply'));
})->done();
```

**Reply by civilCornball** [reply]
Worked - had to change how composer updated my DiscordPHP.
v10 is still in-development, so I chose [Version 10.0.0 Release Candidate 6](https://github.com/discord-php/DiscordPHP/releases/tag/v10.0.0-RC6) by putting this in my composer.json
```php
"require": {
        "team-reflex/discord-php": "10.0.0-RC6"
       }
